sdk/storage/storage-blob/samples/typescript/advanced.ts

```diff
@@ -14,7 +14,7 @@ import {
   ContainerClient,
   BlobServiceClient,
   StorageClient
-} from "../.."; // Change to "@azure/storage-blob" in your package
+} from "../../src"; // Change to "@azure/storage-blob" in your package
```

Comment:
@jeremymeng 
Compilation with `tsc` fails if the exact path "src", is not provided.

Reply:
interesting. `ts-node` has no problem running it.

Reply:
Update - 
package.json -  "build:samples": "cd samples && tsc -p ."

When the above script is executed as part of "preLaunchTask", it fails when the exact path "src" is not provided.